openmoc/compatible/opencg_compatible.py

```diff
@@ -753,8 +768,8 @@ def get_openmoc_lattice(opencg_lattice):
   universes = opencg_lattice.universes
 
   # Initialize an empty array for the OpenMOC nested Universes in this Lattice
-  universe_array = np.ndarray(tuple(np.array(dimension[0:2])), \
-                              dtype=openmoc.Universe)
+  # The Lattice universe array is stored x-y-z in OpenCG, but z-y-x in OpenMOC
```

Comment:
On a related note though, OpenCG's `dimension` attribute is stored as x-y-z since this (may) make more intuitive sense to the user. Hence the need to reverse this array here when initializing OpenMOC's universes array.